storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -6274,6 +6302,17 @@ xb_export_cfg_write_header(
     return (false);
   }
 
+  if (cfg_version >= IB_EXPORT_CFG_VERSION_V5) {
```

Comment:
Can you help me understand this one?
INSTANT is a method of ALTER, why will this be related to EXPORT?

Reply:
After adding INSTANT column in a table. Server add that information during export and verify that during import. So replicating similar behaviour.

Reply:
what about versions 3  & 4 ? before the fix, we are handling until 2?